service-api/module/Application/src/Helpers/ServiceAvailabilityHelper.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers;
+
+use Application\Model\Entity\CaseData;
+use Application\Fixtures\DataWriteHandler;
+use Application\Sirius\EventSender;
+use Psr\Log\LoggerInterface;
+
+class ServiceAvailabilityHelper
+{
+    /**
+     * @psalm-suppress PropertyNotSetInConstructor
+     */
+    protected array $availableServices;
+
+    protected array $processedMessages = [];
+
+    public function __construct(
+        protected array $services,
+        protected CaseData $case,
+        protected array $config
+    ) {
+        $this->processGlobalServicesSettings();
+        $this->mergeServices($this->config);
+    }
+
+    private function processGlobalServicesSettings(): void
+    {
+        $processedGlobalServices = [];
+
+        if ($this->services['EXPERIAN'] !== true) {
+            $processedGlobalServices['EXPERIAN'] = false;
+            $processedGlobalServices['DRIVING_LICENCE'] = false;
+            $processedGlobalServices['PASSPORT'] = false;
+            $processedGlobalServices['NATIONAL_INSURANCE_NUMBER'] = false;
+            $processedGlobalServices['POST_OFFICE'] = $this->services['POST_OFFICE'];
+        } else {
+            foreach ($this->services as $key => $value) {
+                $processedGlobalServices[$key] = $value;
+            }
+        }
+
+        if (
+            $processedGlobalServices['DRIVING_LICENCE'] === false ||
+            $processedGlobalServices['PASSPORT'] === false ||
+            $processedGlobalServices['NATIONAL_INSURANCE_NUMBER'] === false
+        ) {
+            $this->processedMessages['service_status'] = "Some identity verification methods are not presently" .
+                " available";
+        }
+
+        if (
+            $processedGlobalServices['DRIVING_LICENCE'] === false &&
+            $processedGlobalServices['PASSPORT'] === false &&
+            $processedGlobalServices['NATIONAL_INSURANCE_NUMBER'] === false
+        ) {
+            $processedGlobalServices['EXPERIAN'] = false;
+            $this->processedMessages['service_status'] = "Online identity verification is not presently available";
+        }
+
+        if (array_key_exists('message', $this->services)) {
+            $this->processedMessages[] = $this->services['message'];
+        }
+
+        $this->services = $processedGlobalServices;
+    }
+
+    public function getProcessGlobalServicesSettings(): array
+    {
+        return $this->availableServices;
+    }
```

Comment:
I've been told we need to be including the type of the array being returned like: 
```
/**
 * @return array<string, string>
 */
```
or if just a basic indexed array:
```
/**
 * @return string[]
 */
```